app/controllers/files_controller.rb

```diff
@@ -16,6 +16,11 @@ def index
         @last_accessioned_version = Preservation::Client.objects.current_version(params[:item_id])
       rescue Preservation::Client::NotFoundError
         return render status: :unprocessable_entity, plain: "Preservation has not yet received #{params[:item_id]}"
+      rescue Preservation::Client::Error => e
+        message = "Preservation client error getting current version of #{params[:item_id]}: #{e}"
+        Rails.logger.error(message)
+        Honeybadger.notify(message)
+        return render status: :internal_server_error, plain: message
```

Comment:
Is this the right status here? Or should it also be a 422? Or... ?

Reply:
I think this is fine.